app/api.py

```diff
@@ -65,3 +78,15 @@ def update(req: UserCreateRequest, token: str = Depends(get_auth_token)):
     # print(req)
     model.update_user(token, req.user_name, req.leader_card_id)
     return {}
+
+
+# ルーム作成
+@app.post("/room/create", response_model=RoomCreateRequest)
+def room_create(req:RoomCreateRequest,token: str = Depends(get_auth_token)):
+    user = model.create_room(token, req.live_id, req.elect_difficulty)
+
+#ルーム検索
+@app.post("/room/list", response_model=RoomListRequest)
```

Comment:
response_modelにはレスポンスに使うクラスを指定してください (～Responseの方)
```suggestion
@app.post("/room/list", response_model=RoomListResponse)
```